### DISCUSSION TITLE
what is SparseCtrl??

### DISCUSSION THREAD

**Discussion by TigerHH6866** [Mikubill/sd-webui-controlnet#2680] [Q&A] [answered]
1.1.441

a new model??
what fuction?

**Comment by ostap667inbox** [comment]
There is same 3 preprocessors as in 'Scribble/Sketch'

**Comment by huchenlei** [comment]
Adding PR author @continue-revolution to the thread.

**Comment by continue-revolution** [comment, marked as answer]
AnimateDiff only, ignore that if you don’t have AnimateDiff, otherwise read the [doc](https://github.com/continue-revolution/sd-webui-animatediff/blob/master/docs/features.md#controlnet-v2v) in my repo

**Reply by continue-revolution** [reply]
There is no way I can implement it separately from my repo, unfortunately.
